PowerControlPanel/Classes/StartUp/Resources/StringResources.ru.xaml

```diff
@@ -2,127 +2,127 @@
+    <system:String x:Key="Enable">Включить</system:String>
+    <system:String x:Key="Disable">Выключить</system:String>
+    <system:String x:Key="Show">Показать</system:String>
+    <system:String x:Key="Hide">Скрыть</system:String>
+    <system:String x:Key="Close">Закрыть</system:String>
+    <system:String x:Key="Open">Открыть</system:String>
+    <system:String x:Key="Auto">Авто</system:String>
     <!-- Tile Homepage -->
     <system:String x:Key="TileHomePage_TDP">TDP</system:String>
-    <system:String x:Key="TileHomePage_TDPBoost">TDP Boost</system:String>
-    <system:String x:Key="TileHomePage_TDPSustained">TDP Sustained</system:String>
-    <system:String x:Key="TileHomePage_MaxCPU">Max CPU Freq</system:String>
-    <system:String x:Key="TileHomePage_ActiveCores">Active Cores</system:String>
-    <system:String x:Key="TileHomePage_Brightness">Brightness</system:String>
-    <system:String x:Key="TileHomePage_Volume">Volume</system:String>
-    <system:String x:Key="TileHomePage_GPUCLK">GPU Clock</system:String>
-    <system:String x:Key="TileHomePage_Display">Display</system:String>   
-    <system:String x:Key="TileHomePage_Resolution">Display Resolution</system:String>
-    <system:String x:Key="TileHomePage_Scaling">Display Scaling</system:String>
-    <system:String x:Key="TileHomePage_Refresh">Display Refresh</system:String>
-    <system:String x:Key="TileHomePage_ActiveProfile">Active Profile</system:String>
-    <system:String x:Key="TileHomePage_FPSLimit">FPS Limit</system:String>
-    <system:String x:Key="TileHomePage_FanSpeed">Fan Speed</system:String>
+    <system:String x:Key="TileHomePage_TDPBoost">Повышенный TDP</system:String>
+    <system:String x:Key="TileHomePage_TDPSustained">Стабильный TDP</system:String>
+    <system:String x:Key="TileHomePage_MaxCPU">Максимальная частота CPU</system:String>
+    <system:String x:Key="TileHomePage_ActiveCores">Активные ядра</system:String>
+    <system:String x:Key="TileHomePage_Brightness">Яркость</system:String>
+    <system:String x:Key="TileHomePage_Volume">Громкость</system:String>
+    <system:String x:Key="TileHomePage_GPUCLK">Частота GPU</system:String>
+    <system:String x:Key="TileHomePage_Display">Дисплей</system:String>   
+    <system:String x:Key="TileHomePage_Resolution">Разрешение дисплея</system:String>
+    <system:String x:Key="TileHomePage_Scaling">Масштаб дисплея</system:String>
+    <system:String x:Key="TileHomePage_Refresh">Частота обновления дисплея</system:String>
+    <system:String x:Key="TileHomePage_ActiveProfile">Активный профиль</system:String>
+    <system:String x:Key="TileHomePage_FPSLimit">Ограничение FPS</system:String>
+    <system:String x:Key="TileHomePage_FanSpeed">Скорость вентилятора</system:String>
 
     <system:String x:Key="TileHomePage_Intel">Intel</system:String>
-    <system:String x:Key="TileHomePage_PlayNite">PlayNite</system:String>
+    <system:String x:Key="TileHomePage_PlayNite">Playnite</system:String>
```

Comment:
Minor: I agree that Playnite is written without a capital N (https://playnite.link/), but probably this change should be made not only for Russian but globally.

Reply:
Agree